content/examples/flatcar-container-linux/_index.md

```diff
@@ -0,0 +1,204 @@
+    chmod +x /usr/local/bin/flatcar-install
+```
+
+Before you build the image, make sure you have `TINKERBELL_HOST_IP` environment variable set, pointing to the Tinkerbell server IP address. This can usually be done by sourcing `envrc`file, generated by Tinkerbell's `setup.sh` script. To load the environment variables from this file, run the following command:
+
+```sh
+source envrc
+```
+
+You should build the image using the following command:
+
+```shell
+docker build -t $TINKERBELL_HOST_IP/flatcar-install .
+```
+
+Then, you should push the image to Tinkerbell registry, so it can be pulled by workers during workflow execution:
+
+```shell
+docker push $TINKERBELL_HOST_IP/flatcar-install
+```
+
+## Preparing Flatcar Container Linux images
+
+For the installation, worker requires access to the Flatcar Container Linux images. Those can be accessed in 2 different ways:
+
+### Use official mirrors
+
+If your workers have internet access, workers should be able to pull the Flatcar images from the official mirror. Please note,
+that this method will be slower than setting up local mirror for the images.
+
+If you don't have dedicated NAT gateway, which you can use for workers, you can make a Tinkerbell server to act as one using
+the following command:
+
+```shell
+sudo iptables -t nat -A POSTROUTING -o enp0s3 -j MASQUERADE
+sudo iptables -A FORWARD -i enp0s3 -j ACCEPT
+```
+
+Where `enp0s3` is the interface name used for Tinkerbell.
+
+Once done, please make sure your workers have `gateway` field configured in their hardware entries. `gateway` field should
+point to Tinkerbell server IP address.
+
+Example snippet of hardware entry with `gateway` defined:
+```json
+"ip_addresses": [{"address": "192.168.50.10", "gateway": "192.168.50.2", "netmask": "255.255.255.0", "address_family": 4}]
+```
+
+### Use local mirror
+
+If your workers do not have internet access or if you want to speed up the provisioning process,
+you can set up local Flatcar mirror, from where the Flatcar images will be pulled by workers.
+The images will be served by nginx running as part of Tinkerbell stack.
+The mirror can be created and updated using official script, which you can find on [GitHub](https://github.com/kinvolk/flatcar-release-mirror).
+
+#### Creating mirror
+
+To prepare the directories for the mirror and download the mirroring script, run the following command:
+
+```sh
+mkdir /var/tinkerbell/nginx/misc/flatcar
+cd /var/tinkerbell/nginx/misc/flatcar
+wget https://github.com/kinvolk/flatcar-release-mirror/raw/master/flatcar-release-mirror.sh && chmod +x flatcar-release-mirror.sh
+```
+
+Then, run the mirror script:
+```shell
+./flatcar-release-mirror.sh --only-files 'flatcar_production_image\|version' --above-version 2344 --logfile /dev/stderr --channels stable
+```
+
+It will download all Flatcar images above version 2344 only from stable channel. This should require downloading around 1.1 GB of data.
```

Comment:
It will grow over time

```suggestion
It will download all Flatcar images above version 2344 only from stable channel. This currently requires downloading around 1.1 GB of data.
```